bindings/python/iota_sdk/types/common.py

```diff
@@ -112,6 +112,11 @@ def opt_int_encoder(value):
     return None
 
 
+def hex_str_decoder(value: str) -> int:
+    """ Parses a given string as a hexadecimal integer """
```

Comment:
Any reason for the extra spaces?
```suggestion
    """Parses a given string as a hexadecimal integer."""
```

Reply:
Just my aesthetic choice. ~~`pep8`~~`pycodestyle` doesn't care either way it seems, but I'm happy to follow existing conventions.